cms_status_message/models.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    default_status_msg_title = {
+        'info': _('Info'),
+        'success': _('Success'),
+        'danger': _('Error'),
+        'warning': _('Warning'),
```

Comment:
I don't want to force users to stick to those types. Potentially you could pass `type_='foo'` to have a custom alert box with class `alert-foo`.